app/models/place_bid.rb

```diff
@@ -56,6 +56,11 @@ def validate_bid_data
   # rubocop:enable Style/IfUnlessModifier, Style/GuardClause
 
   def amount
-    (params[:bid] && params[:bid][:amount]).to_f.round(2)
+    params_amount = params[:bid][:amount]
+
+    # ensures that "1,000" does not save as "1"
+    params_amount = params_amount.gsub(',', '') if params_amount.is_a?(String)
+
+    (params[:bid] && params_amount).to_f.round(2)
```

Comment:
Let's get rid of this later. Removing floats will break a lot of the specs and require a deeper refactor.